src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const arrayOfMen = people.filter(person => {
+    return arguments.length < 2
```

Comment:
what if `century === false` ? then `arguments.length < 2` will return `false `and `Math.ceil(person.died / 100) === century && person.sex === 'm';` will be executed, whereas it's not expected behavior